data/committees.js

```diff
@@ -57,6 +57,20 @@ export default [
 		],
 		introImage: '/images/committees/studio/studio_motif_applied.png',
 		mission: 'Our mission is to teach skills revolving around video game development such as computer science, game design and art in order to help usher students into the game development industry.',
+		internship: {
+			whatWeDo: 'Studio wants to create a community of game developers, whether you’re interested in gameplay programming, graphics, animation, pixel, music, whatever. We want to help you express yourself through video game development.',
+			whatInternsWillDo: (
+			<ul>
+				<li> Marketing interns will focus on community engagement,
+					particularly in the form of leading socials and increasing member retention</li>
+				<li> Unity Tutorial Subcommittee interns will help host Unity tutorials and bugfix in winter quarter.
+					Unity experience is heavily valued but not absolutely required, as much of game development is a
+					&apos;learn as you go&apos; style. C# skills and teaching experience are appreciated but are soft
+					expectations.</li>
+			</ul> ),
```

Comment:
Let's disable the eslint rule enforcing max line length, either in the file entirely or in front of each relevant line. This is going to destroy our ability to preserve meaningful commits.